recipients.go

```diff
@@ -0,0 +1,70 @@
+package updown
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// Recipients represents a end point type called by Updown on any event
+type Recipient struct {
+	ID    string `json:"id,omitempty"`
+	TYPE  string `json:"type,omitempty"`
+	VALUE string `json:"value,omitempty"`
+}
+
+// CheckService interacts with the checks section of the API
+type RecipientService struct {
+	client *Client
+	cache  Cache
+}
+
+// List lists all the recipients
+func (s *RecipientsService) List() ([]Recipients, *http.Response, error) {
```

Comment:
I believe there is a compilation error here as you declared `RecipientService` not `RecipientsService`